lib/datadog/ci/contrib/cucumber/configuration/settings.rb

```diff
@@ -21,7 +22,9 @@ class Settings < Datadog::CI::Contrib::Settings
 
             option :service_name do |o|
               o.type :string
-              o.default { Datadog.configuration.service_without_fallback || Ext::DEFAULT_SERVICE_NAME }
+              o.default do
+                Datadog.configuration.service_without_fallback || Utils::Git.repository_name || Ext::DEFAULT_SERVICE_NAME
```

Comment:
Should this come from a function provided by our CI gem's utils, instead of a per-framework "implementation"?

eg: `Utils::Something.get_service_name()`?

Reply:
yes, I agree, the logic to look into Datadog.configuration and Utils::Git should be extracted, will do it